app/(authorized)/_layout.tsx

```
@@ -1,26 +1,35 @@
+import { useThemeColor } from "@/components/Themed";
 import { useAuthContext } from "@/context/auth.context";
 import { routes } from "@/utils/utils";
+import { Feather } from "@expo/vector-icons";
 import { Redirect, Stack } from "expo-router";
 import { FC } from "react";
-import { ActivityIndicator } from "react-native";
+import { ActivityIndicator, Pressable } from "react-native";
 
 type LayoutPropsType = {};
 
 const Layout: FC<LayoutPropsType> = ({}) => {
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)